test/test_external.py

```diff
@@ -384,6 +384,11 @@ def check_dataset(config, readme_examples):
         ]
 
 
+def test_load_blocks_with_default_values():
```

Comment:
I would check at least some of the values in the config. Simply omitting the values from the new config would also make the test pass but that's not what we want

Reply:
Yup good point, will do